app/notifications/validators.py

```diff
@@ -57,31 +57,43 @@ def check_service_over_api_rate_limit(service, key_type):
                 raise RateLimitError(rate_limit, interval, key_type)
 
 
-def check_service_over_daily_message_limit(service, key_type):
+def check_service_over_daily_message_limit(service, key_type, notification_type=None):
```

Comment:
hmmm, is it better if we always require `notification_type` and then always check both cache keys (in the same way that we incremement both cache keys).

Reply:
Removed all of the functional code that was tracking the new rate limits - so this is just adding the columns and returning the fields now 👍